### DISCUSSION TITLE
Problem with mobile dogehouse view

### DISCUSSION THREAD

**Discussion by the-rishaw** [benawad/dogehouse#2382] [General]
We have seen a problem in dogehouse in mobile. 
![Screenshot_2021-04-27-15-59-35-31](https://user-images.githubusercontent.com/72737691/116227673-0829ce80-a772-11eb-927d-6e9a1ae17f38.jpg)

**Comment by theotterlord** [comment]
Please create an [issue](https://github.com/benawad/dogehouse/issues), unless one already exists for this problem, in which case, just add your browser version & operating system.

**Comment by dk-raw** [comment]
It is fixed now
